app/cdap/components/RecipeList/ViewAllRecipes.tsx

```diff
@@ -40,6 +44,12 @@ const ViewAllRecipies = () => {
 
   return (
     <>
+      {showRecipesDrawer && (
+        <RecipeDetails
```

Comment:
I have used  DrawerWidget integration in import recipe view.  Below is sample code how it can be consumed. It may give some idea what I've mentioned above.

```javascript 
<DrawerWidget
        anchor="right"
        closeClickHandler={() => {
          toggleOpen(false);
        }}
        headingText={{actionType === VIEW_RECIPE ? T.translate(`${PREFIX}.recipeDetails`) : T.translate(`${PREFIX}.editRecipe`)}
        showBackIcon={selectedRecipe ? true : false}
        onBackIconClick={() => setSelectedRecipe(null)}
        showDivider={false}
        open={isOpen}
        dataTestId={'apply-recipe-drawer-widget'}
        showHeaderSeparator={false}
        children={actionType === VIEW_RECIPE ? renerRecipeDetailsComponent() : renderEditRecipeComponent()}
      />
```

Reply:
Done